tasks/main.yml

```diff
@@ -90,3 +90,5 @@
     - configuration
     - haproxy
     - haproxy-start-enable-service
+
+- meta: flush_handlers
```

Comment:
Thanks I will keep that change to myself. Thanks for quick merge !

On Sep 19, 2016 12:03 PM, "Mischa ter Smitten" <notifications@github.com>
wrote:

> In tasks/main.yml
> <https://github.com/Oefenweb/ansible-haproxy/pull/30#discussion_r79357637>
> :
>
> > @@ -90,3 +90,5 @@
> >      - configuration
> >      - haproxy
> >      - haproxy-start-enable-service
> > +
> > +- meta: flush_handlers
>
> @haad <https://github.com/haad> I rather not have this, since this
> flushes all open handlers. Is there any reason why we would need it?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Oefenweb/ansible-haproxy/pull/30/files/ba0ce0866d24a4649f3f6c450b4eb5824dc15ea3#r79357637>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEIzi6HoWBoh7vhqkY68TfbAsCQn8DJks5qrl3bgaJpZM4KAS-O>
> .
>